package.json

```diff
@@ -26,6 +26,7 @@
     "ima-babel6-polyfill": "^0.12.0",
     "matchmedia-polyfill": "^0.3.0",
     "raf": "^3.3.0",
+    "requestidlecallback": "^0.3.0",
```

Comment:
The shim you have here looks pretty complex. Did you consider this simpler one? https://github.com/HenrikJoreteg/ric-shim